packages/components/src/lib/distributionSpecBuilder.ts

```diff
@@ -255,6 +302,55 @@ export function buildVegaSpec(
           },
         ],
       },
+      {
+        type: "text",
+        name: "announcer",
+        interactive: false,
+        encode: {
+          enter: {
+            x: { signal: String(width), offset: 1 }, // vega would prefer its internal ` "width" ` variable, but that breaks the squiggle playground. Just setting it to the same var as used elsewhere in the spec achieves the same result.
```

Comment:
Btw, I was originally thinking it would be good to just do the number signal in a React overlay, instead of in Vega. Vega can be a pain for things like this; React would give us more flexibility. That said, this is fine for now.